### ISSUE TITLE
Update to Angular 11

### ISSUE THREAD

**Issue by acomanescu** [auth0/auth0-angular#88]
### Describe the problem

Angular is now at version 11. Installing `@auth0/auth0-angular` is only possible with `--force` or `--legacy-peer-deps` arguments.

### What was the expected behavior?

We are expecting a clear install without having to force it.

### Reproduction

> Detail the steps taken to reproduce this error, and whether this issue can be reproduced consistently or if it is intermittent.
> **Note**: If clear, reproducable steps or the smallest sample app demonstrating misbehavior cannot be provided, we may not be able to follow up on this bug report.

1. Generate new Angular project using the latest stable version
2. Execute `npm i @auth0/auth0-angular`
3. 

```
➜ ### git:(master) ✗ npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: @###@1.0.0
npm ERR! Found: @angular/common@11.0.0
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~11.0.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^10.0.2" from @auth0/auth0-angular@1.2.0
npm ERR! node_modules/@auth0/auth0-angular
npm ERR!   @auth0/auth0-angular@"^1.2.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Environment

> Please provide the following:

- **Version of `auth0-angular` used: ^1.2.0**
- **Which browsers have you tested in? CLI**
- **Which version of Angular have you tested with? ~11.0.0**
- **Other modules/plugins/libraries that might be involved: Clean install**

**Comment by stevehobbsdev** [comment]
Thanks for raising @arobert93 - we have closed a PR for this already and anticipate doing a release at some point soon, will notify here when that's out.

https://github.com/auth0/auth0-angular/pull/85

**Comment by frederikprijck** [comment]
@arobert93 This should be resolved using the latest version, 1.3.0. Could you verify ?

**Comment by acomanescu** [comment]
@frederikprijck just checked it and it works as expected now. Thanks!
